tests/test_helpers/dom_helper.ts

```diff
@@ -160,17 +194,13 @@ export function setInputValueAndTrigger(
  *
  * This comment is meant to leave a trace of this change in case some issues were to arise again.
  */
-export async function keyDown(key: string, options: any = {}): Promise<void> {
-  document.activeElement!.dispatchEvent(
-    new KeyboardEvent("keydown", Object.assign({ key, bubbles: true, cancelable: true }, options))
-  );
+export async function keyDown(key: string, options: KeyboardEventInit = {}): Promise<void> {
```

Comment:
If we are bound to specify the target of our event (to make the tests clearer) might as well use triggerkeydown/up everywhere and drop this helper altogether.